libs/sdk-bindings/README.md

```diff
@@ -1,12 +1,16 @@
 # breez-sdk-bindings
 
 This project provides bindings for breez-sdk to various languages.
-Currently supported languges are kotlin & swift.
+Currently supported languges are kotlin, swift and C#.
 For kotlin & swift, we are using [uniffi](https://github.com/mozilla/uniffi-rs).
 
-## prerequisite
-* set the ANDROID_NDK_HOME env variable to your sdk home folder
-* install cargo-ndk to provide simplified android build: ```cargo install cargo-ndk``` 
+## Prerequisites
+* When building for Android:
+  * Set the ANDROID_NDK_HOME env variable to your sdk home folder
+  * Install `cargo-ndk` to provide simplified android build: ```cargo install cargo-ndk``` 
+* When building C# bindings:
```

Comment:
I don't think we need this specific section for C# because there is a target in the make file that does it and you added documentation for it bellow

Reply:
You're right. Done.